docs/src/examples/components/Scrollbar/index.ts

```diff
@@ -0,0 +1,4 @@
+export * as Default from "./Default";
+export * as Horizontal from "./Horizontal";
+export * as Customized from "./Customized";
+export * as FlexLayout from "./FlexLayout";
```

Comment:
The file has a small "l" in the name. Git usually doesn't keep case changes.. You can fix this with `git mv -f Flexlayout.tsx FlexLayout.tsx` then the case change will be tracked by git.

Reply:
Yes correct, I fixed that lint error before I push but seems like it didnt recognised and it didnt took the case changes after that. that command helps me to fix it by running it in the directory --> `cd docs/src/examples/components/Scrollbar` 
Thanks!